src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/Models/Subnet.cs

```diff
@@ -30,13 +30,14 @@ public partial class Subnet : SubResource
         /// <summary>
         /// Initializes a new instance of the Subnet class.
         /// </summary>
-        public Subnet(string id = default(string), string addressPrefix = default(string), NetworkSecurityGroup networkSecurityGroup = default(NetworkSecurityGroup), RouteTable routeTable = default(RouteTable), IList<IPConfiguration> ipConfigurations = default(IList<IPConfiguration>), string provisioningState = default(string), string name = default(string), string etag = default(string))
```

Comment:
@AhmedSalama-MS This is a breaking change.  We would like to avoid breaking changes, can you move the new parameters further down in the list?